src/settings/toml/tests/mod.rs

```diff
@@ -1,8 +1,17 @@
+#[path = "../../../../tests/fixture/wrangler_toml.rs"]
```

Comment:
it's kinda nasty but appears to be the least objectionable way to share testing structs between unit and integration tests.

Reply:
oh wowzas.